packages/web-components/src/dropdown/README.md

```diff
@@ -0,0 +1,176 @@
+# Dropdown
+
+> A dropdown is a user interface component that presents a collapsible list of selectable options to users.
+
+<br />
+
+## **Design Spec**
+
+[Link to Dropdown Design Spec in Figma](https://www.figma.com/file/pI5m9e9hsNkl1odWzxCxNd/Dropdown?node-id=1319-163&t=Hfpd1nwRmumsVKtL-0)
+
+<br />
+
+## **Engineering Spec**
+
+Fluent WC3 DroFluent WC3 Dropdown is a form associated component that extends from the FAST Select [FAST Select](https://explore.fast.design/components/fast-select) and is intended to be as close to the Fluent UI React 9 Dropdown implementation as possible. However, due to the nature of web components there will not be 100% parity between the two.
```

Comment:
I have some strong opinions on how we articulate this.

First, There isn't a concept of "dropdown" as a semantic element or any pattern. I won't argue the name, though it implies appearance and conditional behavior (it doesn't always drop down...) rather than anything semantic. With that said, it is for all intents and purposes a Select which would then extend FAST Select. I don't know enough about the FUIR9 Dropdown as I haven't dug in - but is it actually possible to be closely aligned or are we simply stating that?

Reply:
FUIR9 defines 3 separate components:
`Select` - renders native select
`Dropdown` - allows multi-select and rich formatting of options
`Combobox` - allows typing into the input field to support searching/filtering